spec/recipes_spec/books/pl-u-physics/expected_output.xhtml

```diff
@@ -33320,419 +33320,9 @@ podstaw podane w tre&#x15B;ci zadania warto&#x15B;ci drogi i czasu:<span data-ty
+                                                                                                                                                                                                            
+                                                                                                                                                                                                          </div>
```

Comment:
Here it is visible the change in pl-u-physics test content. I have updated the `expected_output file` within the `actual_outout`.